filechanges/etc/fstab

```diff
@@ -1,5 +0,0 @@
-proc            /proc           proc    defaults          0       0
-/dev/mmcblk0p1  /boot           vfat    defaults,user,rw,umask=000          0       2
-/dev/mmcblk0p2  /               ext4    defaults,noatime  0       1
-# a swapfile is not a swap partition, so no using swapon|off from here on, use  dphys-swapfile swap[on|off]  for that
-/dev/mmcblk0p3  none            swap    sw                0       0
```

Comment:
Right. That's what I have in my installer, but I would expect the existing /etc/fstab to be modified in place, rather than overwriting it, which causes problems when not using a physical Raspberry Pi to boot (or using UUIDs to mount the partitions, instead of hard-coding).